### PR TITLE
DOMA-1613 adde apartment number to contacts table in callcenter, refactoring

### PR DESCRIPTION
**Pull request by akarjakin**

### CODE REVIEW

**Review by nomerdvadcatpyat — APPROVED** [review]

File: apps/condo/domains/common/components/TableFilter.tsx [hunk 1]
```diff
@@ -79,14 +90,41 @@ export const getTextFilterDropdown = (columnName: string, setFiltersApplied: Rea
                 // @ts-ignore
                 value={selectedKeys}
                 onChange={(e) => {
-                    // @ts-ignore
-                    setSelectedKeys(e.target.value)
-                    setFiltersApplied(true)
-                    confirm({ closeDropdown: false })
+                    if (isFunction(setSelectedKeys)) setSelectedKeys(e.target.value)
```
Comment (MrFoxPro) [inline apps/condo/domains/common/components/TableFilter.tsx:93]: Is it ok to shorten this to `setSelectedKeys && setSelectedKeys(e.target.value)`?

File: yarn.lock [hunk 2]
```diff
@@ -565,11 +565,6 @@
   dependencies:
```
Comment (MrFoxPro) [inline yarn.lock:565]: Not sure we need lock file in this pr

File: .gitignore [hunk 3]
```diff
@@ -16,7 +16,6 @@ __media
 _example.js
 codegen.yaml
 schema.graphql
-
```
Comment (MrFoxPro) [inline .gitignore:19]: ?

File: apps/condo/domains/common/components/TableFilter.tsx (hunk 1 above)
Comment (akarjakin) [reply apps/condo/domains/common/components/TableFilter.tsx:93]: actually we want to check if setSelectedKeys not only exists, bat also is a function, that's why I always recommend using such checks in places, where code can throw exception.

File: .gitignore (hunk 3 above)
Comment (akarjakin) [reply .gitignore:19]: it's after yarn install, anyway someone will merge this sooner or later :)

File: yarn.lock (hunk 2 above)
Comment (akarjakin) [reply yarn.lock:565]: it's after yarn install, anyway someone will merge this sooner or later :)

File: .gitignore (hunk 3 above)
Comment (akarjakin) [reply .gitignore:19]: I added submodules here, then removed (not needed actually), and looks like I just removed meaningless empty row too.

File: yarn.lock (hunk 2 above)
Comment (akarjakin) [reply yarn.lock:565]: looks like it's better to commit yarn.lock https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for
